crates/matrix-sdk/src/room/common.rs

```diff
@@ -817,6 +819,65 @@ impl Common {
         let request = delete_tag::v3::Request::new(&user_id, self.inner.room_id(), tag.as_ref());
         self.client.send(request, None).await
     }
+
+    /// Sets whether this room is a DM.
+    ///
+    /// When setting this room as DM, it will be marked as DM for all active
+    /// members of the room. When unsetting this room as DM, it will be
+    /// unmarked as DM for all users, not just the members.
+    ///
+    /// Returns the [`set_global_account_data::v3::Response`] from the server.
+    ///
+    /// # Arguments
+    /// * `is_direct` - Whether to mark this room as direct.
+    pub async fn set_is_direct(&self, is_direct: bool) -> Result<()> {
```

Comment:
The docs describe a different return value than given. I'd prefer to have the documented API return.